src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1243,10 +1243,12 @@
 			else
 				value = "EditAndContinue"
 			end
+		elseif cfg.symbols == p.OFF then
+			value = ""
```

Comment:
This isn't right; `symbols 'off'` will not emit any XML. In VS, the default for symbols is `ProgramDatabase`...
It should be `value = "None"`, and if `symbols` is never called, only then should the element not be written.